ztunnel-1.24.yaml

```
@@ -0,0 +1,75 @@
+package:
+  name: ztunnel-1.24
+  version: 1.24.0
+  epoch: 0
+  description: The `ztunnel` component of istio ambient mesh.
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    provides:
+      - ztunnel=${{package.full-version}}
+    runtime:
+      - ca-certificates-bundle
+      - ip6tables
+      - iptables
+      - libmnl
+      - libnetfilter_conntrack
+      - libnfnetlink
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - cargo-auditable
+      - clang
+      - cmake
+      - openssf-compiler-options
+      - protoc
+      - rust
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/istio/ztunnel
+      tag: ${{package.version}}
+      expected-commit: 1226c1b35f50938f428c71f7dcad3602ea991675
+
+  - runs: |
+      export ZTUNNEL_BUILD_buildStatus=clean
+      export ZTUNNEL_BUILD_buildGitRevision=$(git rev-parse HEAD)
+      export ZTUNNEL_BUILD_buildTag=$(git describe --tags --abbrev=0)
+      export ZTUNNEL_BUILD_buildVersion=$(git rev-parse HEAD)
+      cargo auditable build --release
+      mkdir -p ${{targets.destdir}}/usr/bin
+      cp out/rust/release/ztunnel ${{targets.destdir}}/usr/bin/ztunnel
+
+  - uses: strip
+
+subpackages:
+  - name: ${{package.name}}-compat
```

Should this be instead the following and the omit `dependencies` section entirely?
```suggestion
  - name: ztunnel-compat
```

---

I think this is going to be the default so this will be very much a version stream package as starting 1.25 version of istio so the current version stream format will be more future proof IMHO. 💭